content/mqtt/index.textile

```diff
@@ -0,0 +1,103 @@
+var command = decoder.decode(message.data);
+```
+
+In the above example, @command@ will now contain the message in its original string form.
+
+h2(#use-cases). Recommended usage
+
+We recommend you use MQTT to interact with Ably when:
+* Using devices which do not have a native Ably client library, such as Arduino platforms, C/C++ applications, and so on
+* If you need to work with a very low bandwidth, or want to keep your network traffic to a minimum
+
+h2(#supported). Supported features
+
+MQTT is only able to make use of the basic Realtime pub/sub mechanics of the Ably system. For all other features, such as presence, history and push notifications, you'll need to make use of an Ably client library. We will soon be allowing for devices to be present in channels when connected through MQTT.
+
+If you need to make use of any of these features, we reccomend that you make use of the "Ably REST API":https://www.ably.io/documentation/rest-api.
+
+h2(#security). Security
+
+We recommend that you always make use of "token authentication":https://www.ably.io/documentation/general/authentication#token-authentication over "basic authentication":https://www.ably.io/documentation/general/authentication#basic-authentication when trying to connect from devices you may not trust. In addition, if you're using basic authentication you'll be required to use SSL to connect to our adapter to ensure the API key cannot be accessed by someone listening in.
+
+h3(#token-auth). Token authentication
+
+For token authentication you'll be required to "create a token from your own servers":https://www.ably.io/documentation/general/authentication#token-authentication. Once you have the token, you can simply pass it through when trying to connect to Ably as the connection's @username@ and leaving the @password@ empty. In the future we will be adding "JWT":https://jwt.io/ (JSON Web Tokens) as a form of authentication.
+
+If using token authentication, you can subscribe to a special topic, *[mqtt]tokenevents*, to get a warning when the current connection's token is about to expire. This will be a single message, sent 30 seconds before the token expires, with the 13 byte payload *expirywarning*.
+
+On receiving this, the client is recommended to get a new token, then disconnect and reconnect with the new token themselves. If this is not done, once the token expires, the server will abruptly disconnect the connection.
+
+h3(#ssl). SSL
+
+We support both ssl and non-ssl connections (which use a different port, see above), but strongly recommend using ssl wherever possible. If you are not using ssl, note that the same restrictions apply as if you were using an Ably client without ssl. That is, connection attempts using basic authentication (i.e. an api key) are disallowed, and any namespaces which you've enabled 'require TLS' on will be inaccessible. If this is a problem for you, please do let us know at support@ably.io and we'll see what we can do.
```

Comment:
This section was copied from https://support.ably.io/support/solutions/articles/3000074906-using-the-mqtt-protocol-adapter but the inline links were dropped, could you add them back